testutils/util/websockets.py

```diff
@@ -39,16 +42,21 @@ async def connect():
                 self.url, extra_headers=self.headers, ssl=ssl_context
             )
 
-        attempts = 5
+        attempts = 15
+        sleep_seconds = 15
         while True:
             try:
                 asyncio.get_event_loop().run_until_complete(connect())
                 break
             except websockets.InvalidStatusCode:
                 if self.retry_connect and attempts > 0:
                     attempts -= 1
-                    time.sleep(5)
+                    logger.info(
+                        "websockets: %d retrying on InvalidStatusCode" % attempts
+                    )
+                    time.sleep(sleep_seconds)
                 else:
+                    logger.info("websockets: out of retryies on InvalidStatusCode")
```

Comment:
Nitpick: **retries**

Reply:
I really need to get a dictionary. thanks!